src/time.ts

```diff
@@ -259,13 +253,19 @@ export class CronTime {
 					Object.keys(this.dayOfWeek).length !== 7
 				)
 			) {
+				const preDayChangeDate = date;
 				date = date.plus({ days: 1 });
 				date = date.set({ hour: 0, minute: 0, second: 0 });
 
-				if (this._forwardDSTJump(0, 0, date)) {
-					const [isDone, newDate] = this._findPreviousDSTJump(date);
-					date = newDate;
-					if (isDone) break;
+				if (offset !== date.offset) {
+					offset = date.offset;
+					if (
+						date.hour - preDayChangeDate.hour > 1 ||
+						(date.hour === 1 && preDayChangeDate.hour === 23)
+					) {
+						// We skipped past a valid execution time and must execute immediately
+						break;
+					}
```

Comment:
Why is the second check necessary? Wouldn't an offset change always mean a DST forward jump?

Also, maybe add back a `_forwardDSTJump` with a signature like `(preChangeDate: DateTime, date: DateTime, minutes?: boolean): boolean` to avoid duplication?

Reply:
Not saying it isn't necessary per say, just that I can't seem to wrap my head around it and would like to understand haha

Reply:
resolving as outdated; I went with a different method here